src/app/docs/input-box/input-box-docs.component.html

```diff
@@ -0,0 +1,213 @@
+<sky-docs-demo-page
+  moduleName="SkyInputBoxModule"
+  moduleSourceCodePath="src/app/public/modules/input-box/"
+  pageTitle="Input box"
+>
+  <sky-docs-demo-page-summary>
+    The Input box component provides styling for most prompts to enter data in forms. While this component is compatible with SKY UX's default theme, it provides more styling in modern theme.
```

Comment:
What do you guys think of `Input boxes provide styling for prompts to enter data on forms. They are designed for SKY UX's modern theme but are compatible with the default theme.`?

Denise, I erred on the side of keeping the "SKY UX's" given that the modern theme is still so new. I think this is something we should probably clean up at some point like you suggest, but I'd keep it for now. Just my 2 cents though.

Also, do we need to use a restricted view component to create two versions of this content? It seems like this is mostly geared toward modern theme and that we should use restricted view to make that clear. For example, if we don't want anyone but modern theme consumers to use this component, we should probably make add an opening paragraph like `Input boxes are designed for SKY UX's new modern theme that is only available to internal Blackbaud developers. It is not intended for external use.` Or something like that. Then we could wrap everything else with the restricted view component so that external users and Blackaud users who aren't logged in won't see the rest of the content. ... We may want to handle this differently than I'm suggesting, but I think we definitely need to figure out something along these lines.

Reply:
I updated the intro based on my suggested text in the first paragraph of my comment since no one replied either way.

I'm leaving the conversation unresolved though for the question in the last paragraph. How are we handling the restricted view component for input box? @Blackbaud-AlexKingman, I know you've been working on restricted view. Did you get it set up so that we can just hide input box entirely on the site so that it only shows for logged-in Blackbaud users? If so, I think we can disregard my suggestion in the last paragraph because it was based on how the library previously worked. But if not, we should coordinate on this. Just let me know.

Reply:
Yes, it will only show for logged in Blackbaud users. As soon as we get the final PR through, I can implement it in the service. It will not affect this PR though, and shouldn't hold it up.

Reply:
Sounds good. Resolving this conversation since its sounds like we'll handle the restricted view elsewhere. Thanks!